package.json

```diff
@@ -1,41 +1,56 @@
 {
   "name": "bless",
-  "description": "CSS Post-Processor",
+  "version": "4.0.0-alpha.0",
+  "description": "CSS Post-Processor that chunks CSS files per IE9 selector count restrictions",
   "url": "http://blesscss.com",
   "keywords": [
     "css",
     "parser",
-    "bless",
     "less",
     "sass",
-    "stylus"
+    "stylus",
+    "ie9",
+    "selector-limit"
   ],
   "author": "Paul Young <paulyoungonline+bless@gmail.com>",
-  "version": "4.0.0",
+  "maintainers": [
+    "Matt Smith <mtscout6+bless@gmail.com>",
+    "Alex Abenoja <abenojaalex@gmail.com>"
+  ],
+  "repository": "BlessCSS/bless",
+  "license": "MIT",
   "bin": {
     "blessc": "./bin/blessc"
   },
-  "main": "./lib/bless",
+  "main": "./lib/index",
```

Comment:
No there isn't, that's intentional since the lib folder is the transpiled version of the src folder. It is included in the npm distribution.